ext/sodium/libsodium.c

```diff
@@ -2794,6 +2808,73 @@ PHP_FUNCTION(sodium_compare)
 }
 #endif
 
+#ifdef HAVE_AESGCM
+PHP_FUNCTION(sodium_crypto_aead_aes256gcm_keygen)
```

Comment:
Since this hasn't been done in other functions from this extension yet, can this be merged as-is, and then a dedicated commit will take care of adding `zend_parse_parameters_none()` everywhere? (similar to https://github.com/jedisct1/libsodium-php/commit/0b471123a5fead2037c8ff8cb500e2bd92466216 in the standalone extension)